authorizations/whitelist/src/contract.rs

```diff
@@ -0,0 +1,64 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{Addr, Binary, CosmosMsg, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdResult, to_binary};
+use cw2::set_contract_version;
+use cw_auth_manager::msg::{IsAuthorizedResponse, QueryMsg};
+
+use crate::error::ContractError;
+use crate::msg::{ExecuteMsg, InstantiateMsg};
+use crate::state::{Config, CONFIG};
+
+const CONTRACT_NAME: &str = "crates.io:whitelist";
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn instantiate(
+    deps: DepsMut,
+    _env: Env,
+    _info: MessageInfo,
+    _msg: InstantiateMsg,
+) -> Result<Response, ContractError> {
+    set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
+    let empty: Vec<Addr> = vec![];
+    let config = Config { allowed_senders: empty };
+    CONFIG.save(deps.storage, &config)?;
+    Ok(Response::default().add_attribute("action", "instantiate"))
+}
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn execute(
+    deps: DepsMut,
+    _env: Env,
+    _info: MessageInfo,
+    msg: ExecuteMsg,
+) -> Result<Response, ContractError> {
+    match msg {
+        ExecuteMsg::Allow { addr } => {
+            CONFIG.update(deps.storage, |mut config| -> Result<Config, ContractError>{
```

Comment:
Also will want to do some permissioning once this is more fleshed out here. Need to answer: who can add a new allowed sender?

Reply:
> Map<Addr, cosmwasm_std::Empty>

Yeah, 100%. Wrote this very quickly to test/demonstrate the approach worked. 

>  Need to answer: who can add a new allowed sender?

I think those semantics will be difficult to pin down. In this case (whitelist contract and similar) it will come down to what the author of the authorization wants. In the auth-manager, we may want to have some more explicit rules.

Reply:
Fixed this based on your suggestions. Added a check so that only the dao can add `Addr`s to the whitelist. This can be managed differently by each authorization contract.